index.html

```diff
@@ -2477,10 +2477,11 @@ <h3>Data Schemas</h3>
 data syntax, nor enables the definition of arbitrary encodings to alternate
 representation formats.
         </p>
-
+       
         <p>
 This specification defines the following <a>property</a> for the expression of a
-data schema:
+data schema, which MAY be included by an <a>issuer</a> in  
```

Comment:
I don't disagree with this statement, but keep in mind that non-testable MAY statements are effectively worthless -- you can't assert conformance either way. Sure, someone MAY do this, but they may also do many other things that are unstated. I suggest just changing this from "MAY" to "can" -- don't use normative language for stuff that's not testable.

Reply:
If we remove this, we can downgrade this to a V1.1 change as well.

Reply:
Happy to replace MAY with can. I thought that because MAY does not affect conformance then the edit would qualify as v1.1.

Reply:
My general thinking here is that because we're adding normative language that it would make it a V1.2 change due to the following clause in [6.1.3 class 3](https://www.w3.org/2020/Process-20200915/#correction-classes)

```
clears up an ambiguity or under-specified part of the specification in such a way that data, a processor, or an agent whose conformance was once unclear becomes clearly either conforming or non-conforming.
```

Without the normative keyword addition, I generally see it as following under the following statement instead:

```
clarifying informative use cases or other non-normative text
```

Reply:
```suggestion
data schema, which can be included by an <a>issuer</a> in  
```